### ISSUE TITLE
windows下训练loss不下降，

### ISSUE THREAD

**Issue by guanyonglai** [chenyuntc/pytorch-best-practice#11]
因为我在Python3运行，所以要做一些小的修改，，
win10-64、CPU环境，
1.utils/visualize.py 44行：win=unicode(name) --> win=str(name)
2.main.py 22行：  加 import config
3.main.py 108行：loss_meter.add(loss.data[0]) -->  loss_meter.add(loss.item())
4.config.py 10行：load_model_path = 'checkpoints/model.pth' --> load_model_path = None
5.config.py 12行：batch_size = 128 --> batch_size = 8
6.config.py 21行：lr = 0.1 --> lr = 0.001
7.config.py 31行：for k,v in kwargs.iteritems() --> for k,v in kwargs.items()
8.没有执行python -m visdom.server，配置好路径之后直接 python main.py train
打印出loss格式如下，发现loss一直在0.6-1.5之间浮动：
loss: tensor(0.7035, grad_fn=<NllLossBackward>)
也出现了别的同学说的准确率一直在50%左右，也就是学了跟不学一样，
